<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="serial_monitor.qrc">
    <normaloff>:/serialmonitor/win32/serial_monitor.ico</normaloff>:/serialmonitor/win32/serial_monitor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextBrowser" name="receivedDataView">
      <property name="placeholderText">
       <string>Received Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveReceivedData"/>
    <addaction name="separator"/>
    <addaction name="actionExitApplication"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutApplication"/>
   </widget>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Port</string>
    </property>
    <addaction name="actionLoadSerialPorts"/>
    <addaction name="actionToggleConnection"/>
    <addaction name="actionSendData"/>
    <addaction name="separator"/>
    <addaction name="actionPortSettings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewReceivedAsAscii"/>
    <addaction name="actionViewReceivedAsHex"/>
    <addaction name="separator"/>
    <addaction name="actionClearReceivedData"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuPort"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="tbSerialPort">
   <property name="windowTitle">
    <string>Serial Port Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleConnection"/>
   <addaction name="actionLoadSerialPorts"/>
   <addaction name="actionPortSettings"/>
  </widget>
  <widget class="QToolBar" name="tbSendData">
   <property name="windowTitle">
    <string>Send Data Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSendData"/>
  </widget>
  <action name="actionExitApplication">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/exit.png</normaloff>:/serialmonitor/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/qt_logo.png</normaloff>:/serialmonitor/icons/qt_logo.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAboutApplication">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/serial_monitor.png</normaloff>:/serialmonitor/icons/serial_monitor.png</iconset>
   </property>
   <property name="text">
    <string>About Serial Monitor</string>
   </property>
   <property name="toolTip">
    <string>About Serial Monitor</string>
   </property>
  </action>
  <action name="actionToggleConnection">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/connect.png</normaloff>:/serialmonitor/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F6</string>
   </property>
  </action>
  <action name="actionLoadSerialPorts">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/arrow_circle.png</normaloff>:/serialmonitor/icons/arrow_circle.png</iconset>
   </property>
   <property name="text">
    <string>Load serial ports</string>
   </property>
   <property name="toolTip">
    <string>Load serial ports</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionPortSettings">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/gear.png</normaloff>:/serialmonitor/icons/gear.png</iconset>
   </property>
   <property name="text">
    <string>Port Settings</string>
   </property>
   <property name="toolTip">
    <string>Port Settings</string>
   </property>
  </action>
  <action name="actionSendData">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/arrow.png</normaloff>:/serialmonitor/icons/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Send Data</string>
   </property>
   <property name="toolTip">
    <string>Send Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClearReceivedData">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/broom.png</normaloff>:/serialmonitor/icons/broom.png</iconset>
   </property>
   <property name="text">
    <string>Clear Received Data</string>
   </property>
   <property name="toolTip">
    <string>Clear Received Data</string>
   </property>
  </action>
  <action name="actionSaveReceivedData">
   <property name="icon">
    <iconset resource="serial_monitor.qrc">
     <normaloff>:/serialmonitor/icons/disk.png</normaloff>:/serialmonitor/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save Received Data</string>
   </property>
   <property name="toolTip">
    <string>Save Received Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionViewReceivedAsHex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Received as HEX</string>
   </property>
  </action>
  <action name="actionViewReceivedAsAscii">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Received as ASCII</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="serial_monitor.qrc"/>
 </resources>
 <connections/>
</ui>
